templates/index.html

```diff
@@ -2,10 +2,21 @@
 <html>
   <head>
     <title>Play Nim</title>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
   </head>
   <body>
 
-      <h1>So it begins...</h1>
-      {{ state }}
+    <h1>So it begins...</h1>
+
+    <div id="pile0"></div>
+    <div id="pile1"></div>
+    <div id="pile2"></div>
+    <div id="pile3"></div>
```

Comment:
Why add more than 3 piles now?

Reply:
To illustrate that they won't be populated to dave.